x-pack/plugins/fleet/server/services/agent_policy.ts

```diff
@@ -17,14 +16,18 @@ import type {
 } from '@kbn/core/server';
 
 import type { AuthenticatedUser } from '@kbn/security-plugin/server';
+import type { BulkResponseItem } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
 
 import { DEFAULT_SPACE_ID } from '@kbn/spaces-plugin/common/constants';
 
+import { lt } from 'semver';
```

Comment:
(nit) Can we group this dependency with the other "external package" imports above - e.g. below `p-map`?